src/components/validations/validation-icon.spec.js

```diff
@@ -47,7 +60,7 @@ describe('ValidationIcon', () => {
 function renderWithContext(props) {
   return mount(
     <InputPresentationContext.Provider value={ props }>
-      <ValidationIcon type='error' />
+      <ValidationIcon error='Message' type='error' />
```

Comment:
```suggestion
      <ValidationIcon error='Message' />
```

`type` prop is no longer used.